packages/stems/src/components/PillButton/PillButton.tsx

```diff
@@ -0,0 +1,29 @@
+import cn from 'classnames'
+
+import { Button, Type as ButtonType } from 'components/Button'
+
+import styles from './PillButton.module.css'
+import { PillButtonProps, Type as PillButtonType } from './types'
+
+const TYPE_STYLE_MAP = {
+  [PillButtonType.PRIMARY]: styles.primary,
+  [PillButtonType.SECONDARY]: styles.secondary
+}
+export const PillButton = (props: PillButtonProps) => {
+  const {
+    text,
+    onClick,
+    type = PillButtonType.PRIMARY,
+    className,
+    textClassName
+  } = props
+  return (
+    <Button
```

Comment:
How hard would it be to make a `PillRadioButton` that uses a `<input type="radio">` under the hood but keeps all the styles :eyes: